app/models/concerns/defendant_upliftable.rb

```diff
@@ -54,5 +54,13 @@ def defendant_uplift?
     def orphan_defendant_uplift?
       unique_code.in?(self.class.const_get(:ORPHAN_DEFENDANT_UPLIFTS))
     end
+
```

Comment:
I'm ignoring the code climate warnings for now as this code will likely change again when we get t grad fees. Once all fees are calculated, we can refactor this and case_upliftable